src/style.css

```
@@ -1 +1,70 @@
 /* Styles go here */
+* {
+  /* box-sizing: border-box; */
+  padding: 0;
+  margin: 0;
```

```suggestion
  box-sizing: border-box;
```
redundant to reset padding and margins for all selectors.
reset only for elements where it is necessary
* selector in general only for box-sizing property